db/indexes.go

```diff
@@ -458,3 +458,14 @@ func replaceSyncTokensQuery(statement string, useXattrs bool) string {
 func replaceIndexTokensQuery(statement string, idx SGIndex, useXattrs bool) string {
 	return strings.Replace(statement, indexToken, idx.fullIndexName(useXattrs), -1)
 }
+
+// Replace $$activeOnlyFilter placeholder with activeOnlyFilterExpression if activeOnly is true
+// and an empty string otherwise in the channel query statement.
+func replaceActiveOnlyFilter(statement string, activeOnly bool) string {
```

Comment:
As mentioned on the previous review, this should be moved to query.go.

Reply:
The replaceActiveOnlyFilter function has been moved to query.go